src/EnergyPlus/SolarShading.cc

```diff
@@ -5791,23 +5774,22 @@ namespace SolarShading {
 
                     if (OverlapArea > 0.001) {
                         ++JBKS;
-                        if (Surface(BackSurfNum).BaseSurf == BackSurfNum) JBKSbase = JBKS;
```

Comment:
@amirroth Found one sneaky Base Surface reference that relied on sub-surfaces following directly after. Fixes big diffs in Exercise1B-Solution (1 zone, 6 surfaces, nice one to debug). But does not fix the Zone Component Loads Report in 5ZoneAirCooled_ZoneAirMassFlowBalance.